source/configuration_manual/stats/openmetrics.rst

```diff
@@ -43,6 +43,8 @@ These are called ``dovecot_build_info`` and ``process_start_time_seconds``.
 
 .. versionchanged:: 2.3.14 Metric timestamps are dropped.
 
+.. versionchanged:: v2.4.0,v3.0.0 All fields listed in ``fields`` are exported too.
```

Comment:
Does this need to be in upgrading page also?

Reply:
Not imo

Reply:
It does change the existing behavior though, but maybe it doesn't matter.